### DISCUSSION TITLE
*help Not able to install mangodb

### DISCUSSION THREAD

**Discussion by tej4360** [learndevopsonline/learndevopsonline#497] [Q&A]
I am not able to install mango db in my T3 Micro instance,  tried finding the issue but no luck can u please check the screenshot and help.
<img width="572" alt="Screenshot 2023-04-11 at 12 53 29 PM" src="https://user-images.githubusercontent.com/130283102/231088841-c9f87c55-6cc2-485a-8dd5-e00c2ef3b55c.png">

**Comment by tej4360** [comment]
Got issue

**Comment by Manju9876** [comment]
hi @tej4360 , 
since by default the system doesn't have mongodb, so we need to set up the repo file for mongodb by the provided content by the vendor and later you try to install the mongodb-org
